### DISCUSSION TITLE
How to display a model and results?

### DISCUSSION THREAD

**Discussion by xyont** [TLCFEM/suanPan#125] [General]
hello,

after running of example files [truss-roof.supan](https://tlcfem.github.io/suanPan-manual/2.9/Example/Structural/Statics/truss-roof/) i did not know further to work around to display a model and results.

trying with `plot `command shown none to display, only graphic windows with legend. is there any keyword to generate VTK file format output?

thanks in advances.

**Comment by TLCFEM** [comment]
`plot` gives a simple visualisation of the model, which is used to generate the figure shown. I think you are looking for [Recorder](https://tlcfem.github.io/suanPan-manual/2.9/Library/Recorder/Recorder/) to record results.
